<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>344</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>460</width>
    <height>344</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>460</width>
    <height>344</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Internet of Clocks</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pBsync">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>94</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Sincronizar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBalarm">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>60</y>
      <width>94</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Alarma</string>
    </property>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>311</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>460</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCliente">
    <property name="title">
     <string>Cliente</string>
    </property>
    <addaction name="actionConectar"/>
    <addaction name="actionDetectar"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <addaction name="menuCliente"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionConectar">
   <property name="text">
    <string>Conectar</string>
   </property>
  </action>
  <action name="actionDetectar">
   <property name="text">
    <string>Detectar</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
